bundle/python_bundle_workflow/README.md

```diff
@@ -0,0 +1,26 @@
+
+# Write a Bundle workflow in Python
+
+MONAI provides the `BundleWorkflow` interface from version 1.2, which is used as the standard API for other applications to access and operate a bundle workflow, like training, evaluation, or inference. `BundleWorkflow` can help reduce the effort of application development and define standard interface and required properties for bundle development. Once a bundle workflow inherits from `BundleWorkflow`, we can accept it in the associated applications, no matter the bundle workflow is implemented by JSON / YAML or python code.
+
+MONAI already implemented `ConfigWorkflow` for the common JSON / YAML config-based bundle workflow, this example shows how to write python-based training and inference workflows and implement the interfaces: `initialize`, `run`, `finalize`, `_set_property`, `_get_property`, and execute them with bundle CLI command.
+
+## Execute training and inference with bundle script - `run_workflow`
+
+There are several predefined scripts in the MONAI bundle module, here we leverage the `run_workflow` script to execute the python-based workflows in `scripts/train.py` and `scripts/inference.py`.
```

Comment:
Could you please help specify more clearly where is `run_workflow`? Maybe also a link here? https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/scripts.py

Reply:
I think `run_workflow` is not in https://docs.monai.io/en/latest/bundle.html yet, only it's included, I can add the link.

Thanks.